_data/feats.yml

```
@@ -9,3 +69,418 @@ content:
+      unless you attack it. If you succeed on a second Animal Handling check
+      on the creature with a standard action within five minutes of the
+      first check with an increased DC of +6 of the previous check, you gain
+      the creature as an ally for the current combat, or if this is performed
+      out of combat it just leaves you in peace after five minutes. Attacking a
+      convinced creature ends this effect and cannot be performed again for
+      one day.   
+      
+      Tiny:       10
+      
+      Small:      14
+      
+      Medium:     17
+      
+      Large:      22
+      
+      Huge:       26
+      
+      Gargantuan: 31
+      
+  - name: Gleaning Strategist
+    prerequisite: >
+      Have at least five levels in Fighter
+    description: |
+      If you only make one attack this turn (Foregoing extra attacks), you
+      can gain knowledge on the creature you attack and perhaps even provide
+      aid to others in combat against those types of foes. Roll 1d4 to
+      determine the effect.
+      
+      1. Learn a targets HP and AC
+      
+      2. Learn a target's weaknesses and immunities
+      
+      3. Learn a targets standard attack bonus and damage
+      
+      4. All allies and self gain advantage on their next attack against
+      the target + pick one of the other options to learn
+      
+      Whatever the result, you gain a +2 to attack rolls against that creature
+      for the remainder of the encounter. These bonuses do not carry over to
+      additional creatures besides the one that was attacked even if they
+      are the same type of creature.
+      
+  - name: Grasshopper's Repose
+    prerequisite: >
+      Be proficient in Acrobatics
+    description: |
+      You are skilled with adding the attacks of your foes to your own
+      movements. As a reaction, whenever a foe fails an attack roll on you
+      with a melee attack within 5 feet and are of the same size or smaller,
+      you may attempt an Acrobatics Check of DC 16 to flip over your attacker
+      and land on the opposite side of them, gaining advantage on your next
+      attack against them. This can only be performed if there is at least 5
+      feet above the opponent and the other side of them is unoccupied. You
+      end your maneuver 5 feet in the air and land to the floor. This movement
+      does not trigger Attacks of Opportunity. If you fail the check, they
+      get to make an Attack of Opportunity against you with advantage and
+      you do not get to move from your square.   
+      
+  - name: Higher Ground
+    prerequisite: >
+      Have at least one level in Ranger
+    description: |
+      With the presence of a bird of prey, you hunt from up high. You gain a
+      +2 to ranged attack rolls and damage rolls for when you have at least
+      10 feet of height advantage over a foe. These bonuses increase to +3 at
+      10th level.
+      
+   - name: Historian
```

```suggestion
  - name: Historian
```

there is one extra space causing the code to not compile